payments/card/redirect.md

```diff
@@ -21,31 +21,111 @@ simplest and most common way to implement Card Payments." %}
-  your Purchase information. This will generate a payment object with a unique
-  `paymentID`. You will receive a **redirect URL** to a Swedbank Pay payment
-  page.
-* You need to redirect the payer's browser to that specified URL so that she can
-  enter the credit card details in a secure Swedbank Pay environment.
-* Swedbank Pay will handle 3-D Secure authentication when this is required.
-* Swedbank Pay will redirect the payer's browser to - one of two specified URLs,
-  depending on whether the payment session is followed through completely or
-  cancelled beforehand. Please note that both a successful and rejected payment
-  reach completion, in contrast to a cancelled payment.
-* When you detect that the payer reach your `completeUrl`, you need to do a
-  `GET` request to receive the state of the transaction, containing the
-  `paymentID` generated in the first step, to receive the state of the
+Redirect is the integration that lets Swedbank Pay handle the payments, while
+you handle your core activities. When ready to pay, the consumer will be
+redirected to a secure Swedbank Pay hosted site. Finally, the consumer will be
+redirected back to your website after the payment process.
+
+This is how the payment window might look like:
+
+![screenshot of the redirect card payment page][card-payment]{:height="500px" width="425px"}
+
+## Purchase flow
+
+```mermaid
+sequenceDiagram
+    participant Payer
+    participant Merchant
+    participant SwedbankPay as Swedbank Pay
+
+    activate Payer
+    Payer->>-Merchant: start purchase
+    activate Merchant
+    Merchant->>-SwedbankPay: POST /psp/creditcard/payments
+    activate SwedbankPay
+    note left of Payer: First API request
+    SwedbankPay-->>-Merchant: rel: redirect-authorization ①
+    activate Merchant
+    Merchant-->>-Payer: authorization page
+    activate Payer
+    Payer->>-SwedbankPay: access authorization page
+    activate SwedbankPay
+    note left of Payer: redirect to SwedbankPay ②
+    SwedbankPay-->>-Payer: display purchase information
+    activate Payer
+    Payer->>Payer: input creditcard information ③
+    Payer->>-SwedbankPay: submit creditcard information
+        opt Card supports 3-D Secure
+        note left of Payer: Authentication Challenge ④
+        SwedbankPay-->>Payer: redirect to IssuingBank
+        activate Payer
+        Payer->>-IssuingBank: 3-D Secure authentication process
+        activate IssuingBank
+        IssuingBank->>-Payer: 3-D Secure authentication process
+        activate Payer
+        Payer->>-SwedbankPay: access authentication page
+        activate SwedbankPay
+        end
+    SwedbankPay-->>-Payer: CompleteUrl ⑤
+    activate Payer
+    Payer->>-SwedbankPay: GET <payment.id> ⑥
+    activate SwedbankPay
+    note left of Merchant: Second API request
+    SwedbankPay-->>-Merchant: rel: redirect-authorization
+    activate Merchant
+    Merchant-->>-Payer: display purchase result
+        opt Callback is set
```

Comment:
```suggestion

    opt Callback is set
```
Add some whitespace around the opt so when reading the markdown there is a visible division between sections of the sequence diagram